### ISSUE TITLE
Join Channel by link

### ISSUE THREAD

**Issue by bidak** [vysheng/tg#866]
why does import_channel_link give error? (INVITE_HASH_INVALID)
```
import_channel_link https://telegram.me/X
```

**Comment by kamikazechaser** [comment]
Same issue

**Comment by Shatnerz** [comment]
@bidak @kamikazechaser Hey, did you ever find a solution? I am having the same problem

**Comment by kamikazechaser** [comment]
@Shatnerz I actually got it working untill it was was patched late july/early august. tg-cli is also unmaintained and I suggest you to look into alternative CORE libraries.

**Comment by Shatnerz** [comment]
@kamikazechaser Can you explain what alternative CORE libraries are? Assume I know nothing. I see CORE thrown around all the time and it is always something different.

**Comment by kamikazechaser** [comment]
https://github.com/enricostara/telegram.link/

**Comment by kamikazechaser** [comment]
@Shatnerz I agree its a bit misused, however Core is generally referred to the highest level Telegram API. However Telegram classifies both Bot and telegram API inder CORE API

**Comment by Shatnerz** [comment]
@kamikazechaser Nice, I'll take a look. Maybe when I get a chance I can write something for python too. A lot of people were wrapping python around vysheng/tg.

Thanks again
